proposed/event-dispatcher-meta.md

```diff
@@ -37,85 +37,84 @@ The Working Group identified four possible workflows for event passing, based on
+* Passing an object to a series of Listeners to allow it to be modified before it is saved to a persistence system.
+* Passing a collection to a series of Listeners to allow them to register values with it so that the Emitter may act on all of the collected information.
+* Passing a collection to a series of Listeners to allow them to modify the collection in some way before the Emitter takes action.
+* Passing some contextual information to a series of Listeners so that all of them may "vote" on what action to take, with the Emitter deciding based on the aggregate information provided.
+* Passing an object to a series of listeners and allowing one of them to set a value and then prevent further listeners from running.
 
 ### 4.3 Immutable events
 
-Initially the Working Group wished to define all Events as immutable message objects, similar to PSR-7.  However, that proved problematic in the Processor case.  In that case Listeners needed a way to return data to the caller.  In concept there were three possible avenues:
+Initially the Working Group wished to define all Events as immutable message objects, similar to PSR-7.  However, that proved problematic in all but the one-way notification case.  In the other scenarios Listeners needed a way to return data to the caller.  In concept there were three possible avenues:
 
 * Make the event mutable and modify it in place.
 * Require that events be evolvable (immutable but with `with*()` methods like PSR-7 and PSR-13) and that listeners return the event to pass along.
 * Make the Event immutable but aggregate and return the return value from each Listener.
 
-However, Stoppable Task (the alternative chain case) also needed to have a channel by which to indicate that further listeners should not be called.  That could be done either by:
+However, Stoppable Events (the alternative chain case) also needed to have a channel by which to indicate that further listeners should not be called.  That could be done either by:
 
-* Modifying the Task (`stopPropagation()`)
+* Modifying the Event (e.g., calling a `stopPropagation()` method)
 * Returning a sentinel value from the listener (`true` or `false`) to indicate that propagation should terminate.
-* Evolving the Task to be stopped (`withPropagationStopped()`)
+* Evolving the Event to be stopped (`withPropagationStopped()`)
 
-Of those, the first would mandate a mutable Task in at least some cases.  The second would mandate a mutable Task as the return value was already in use.  And the third seemed unnecessarily ceremonial and pedantic.
+Of those, the first would mandate a mutable Event in at least some cases.  The second would mandate a mutable Event as the return value was already in use.  And the third seemed unnecessarily ceremonial and pedantic.
```

Comment:
I think you need to clarify "unnecessarily ceremonial and pedantic": _how_ and _why_ does the WG come to that conclusion? (The phrase is quite subjective, so we need to provide our context.)